languages/csharp/exercises/concept/randomness/.meta/Example.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+
+public class Player
```

Comment:
```suggestion
public static class Player
```

Reply:
Why static?  If everything else is equal (and I think it is here) then is it not better to get the student out of the habit of thinking in terms of static classes?

Reply:
Well, personally I make everything static _unless_ I can't. But maybe that is just me. There is no state here that is linked to a single player instance so that is why I would prefer static. I'll post a comment in our Slack channel to ask for someone else to chime in.

Reply:
I think I agree with Erik on this one. As far as I can tell the only purpose of the class is to generate numbers within a certain range without storing them internally. I don't really see the need to create a Player object for that? Am I missing something here?

Reply:
Leaving aside he debate about the non-static classes I think there are two downsides to making `Player` a static class in this particular exercise:

- We will no longer be teaching the student to favor a static class for `System.Random` which i think is well worth teaching.

- We end up with rather unrealistic code.  Any game is bound to have multiple players and any player is bound to have state and more capabilities.  I don’t think that because we sacrifice this baggage in the interests of simplicity that we should also sacrifice coding realism.

We're not doing much in this exercise.  We should not miss the opportunity for students to get more familiar with instantiated classes.

Reply:
> We will no longer be teaching the student to favor a static class for System.Random which i think is well worth teaching. if everything is static they never have to go through that thought process.

This is a good point.

> We end up with rather unrealistic code. Any game is bound to have multiple players and any player is bound to have state and more capabilities. I don’t think that because we sacrifice this baggage in the interests of simplicity that we should also sacrifice coding realism.

I see what you are saying here. Let's keep it as it is then.